main.go

```diff
@@ -557,7 +557,7 @@ func exportArtifacts(proj xcodeproj.XcodeProj, scheme string, schemeBuildDir str
 		// Find the TARGET_BUILD_DIR for the target
 		var targetDir string
 		{
-			customOptions = []string{"-sdk", simulatorName}
+			customOptions = append(customOptions, []string{"-sdk", simulatorName}...)
```

Comment:
Consider if the customOptions contains -sdk option. Similarly to https://github.com/bitrise-io/steps-xcode-archive/blob/f6f55f7ec9957739c8413f86197e87cfb0576eb5/main.go#L404.